org.ekstep.lessonbrowser-1.4/test/editor/plugin.spec.js

```diff
@@ -1,13 +1,10 @@
 describe("lesson browser plugin", function() {
     var manifest, path, ctrl, $scope, pluginInstance;
     beforeAll(function(done) {
-        CollectionEditorTestFramework.init(function() {
+       org.ekstep.pluginframework.pluginManager.loadPlugin('org.ekstep.lessonbrowser', '1.4', function() {
```

Comment:
If i change the version number, then none of these test cases will work. Take this versions from the list & execute the testcases for all the versions. So that we have verify for backward compatibility in any changes happened.